docs/reference/security.rst

```diff
@@ -252,6 +251,11 @@ And specify it as Sonata security handler on your configuration:
 ACL and FriendsOfSymfony/UserBundle
 -----------------------------------
 
+.. note::
```

Comment:
To me we should remove all references to FOSUserBundle, and replace with references to SonataUserBundle when needed.

References to the install of SonataUserBundle should be pointed to the SonataUserBundle docs.